invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/ComputerTabletCommunitiesItem.js

```diff
@@ -8,10 +8,11 @@ import { i18next } from "@translations/invenio_app_rdm/i18next";
 import _get from "lodash/get";
 import React from "react";
 import { Image } from "react-invenio-forms";
-import { Icon, Item, Label } from "semantic-ui-react";
+import { Icon, Item, Label, Grid } from "semantic-ui-react";
+import { DateTime } from "luxon";
 
 export const ComputerTabletCommunitiesItem = ({ result }) => {
-  const community_type = result.ui?.type?.title_l10n;
+  const community_type = result.metadata.type?.title[i18next.language];
```

Comment:
this is incorrect usage of the translation - the type is already resolved on the backend under .ui as it was implemented before